daemon/api_prompting.go

```diff
@@ -97,6 +100,186 @@ func getUserID(r *http.Request) (uint32, Response) {
 	return uint32(userIDInt), nil
 }
 
+type invalidReason string
+
+const (
+	unsupportedValueReason invalidReason = "unsupported-value"
+	parseErrorReason       invalidReason = "parse-error"
+)
+
+type invalidFieldValue struct {
+	Reason   invalidReason `json:"invalid-reason"`
+	Metadata interface{}   `json:"metadata,omitempty"`
+}
+
+type UnsupportedValueValue struct {
+	err *prompting_errors.UnsupportedValueError
+}
```

Comment:
no need to overly complicate this
```suggestion
type promptingUnsupportedValueError prompting_errors.UnsupportedValueError
```